src/php/ext/grpc/byte_buffer.c

```diff
@@ -16,23 +16,15 @@
  *
  */
 
-#ifdef HAVE_CONFIG_H
-#include "config.h"
+#include "byte_buffer.h"
```

Comment:
Can you just move this line L19 to below L25? Then you don't need the whole block of
```
#ifdef _GNU_SOURCE
  #undef _GNU_SOURCE
#endif
```

Reply:
sooo, the block codes have in php lib, thanks a lot, fixed.